library.json

```
@@ -1,6 +1,6 @@
 {
-  "machineName": "H5P.ThreeSixty",
-  "title": "Three Sixty",
+  "machineName": "H5P.NDLAThreeSixty",
+  "title": "NDL Three Sixty",
```

I don't know if it's necessary to change the title here?

```suggestion
  "title": "NDLA Three Sixty",
```

```suggestion
  "title": "Three Sixty",
```